.github/workflows/local-dev-ci.yaml

```diff
@@ -0,0 +1,60 @@
+name: Execute local test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - "docs/**"
+      - "**/*.md"
+
+jobs:
+  local-dev-test:
+    name: local-dev-test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: v1.17
+
+      - uses: azure/setup-kubectl@v3
+        with:
+          version: v1.23.3
+        
+      - name: Checkout kcp repository
+        uses: actions/checkout@v3
+        with:
+          repository: kcp-dev/kcp
+          ref: release-0.6
```

Comment:
You should probably bump that version to 0.7

Reply:
When PR #192 is merged, I will change that version to 0.7

Reply:
Note: this is not only kcp version that will need to be updated but also kubectl 1.24.x and go v1.18 versions